website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -10,78 +10,121 @@
 - As a layout mechanism.
 - As a replacement for a spreadsheet or similar application.
 
-### Column placement
+## Columns
 
-<Hds::Table>
-    <:head as |H|>
-        <H.Tr>
-            <H.Th>Start</H.Th>
-            <H.Th>Middle</H.Th>
-            <H.Th>End</H.Th>
-        </H.Tr>
-    </:head>
-</Hds::Table>
+### Sorting
+
+![Header column sorting](/assets/components/table/table-sorting.png =1000x*)
+
+- Sorting is not relevant for all content, so consider when to apply sorting thoughtfully.
+- Columns that do contain a sortable data type are interactive and therefore have corresponding hover, active, and focus states.
+- A Table may only be sorted by a single value at a time.
+
+### Width
+
+Column width is determined by manually resizing the header column and cells within Figma. As a best practice, column width should be adjusted to fit the longest data type within the cell.
+
+### Alignment
+
+Use consistent alignment throughout the Table. We recommend using end-alignment in the last column when displaying non-text based content. 
+
+!!! Do
 
-Column placement determines the visual styling based on where the column is placed relatively to other columns in a table header.
+![Table column placement example](/assets/components/table/table-alignment-do.png =500x*)
+!!!
+
+!!! Dont
+
+![Table column placement example](/assets/components/table/table-alignment-dont.png =500x*)
+!!!
+
+### Placement
 
 !!! Info
 
-The column placement property is only relevant within Figma design tooling and doesn’t exist as a property within the production code component.
+The column placement property is only relevant within Figma and doesn’t exist as a property within the code.
+!!!
+
+Column placement determines the visual styling based on where the column is placed relative to other columns in the Table.
+
+![Table column placement example](/assets/components/table/table-col-placement.png =750x*)
+
+## Rows
+
+### Striping
 
+![Table striping examples](/assets/components/table/table-striping.png =810x*)
+
+While striping is not required, we recommend it for the added usability benefits. 
+
+When using striping in a Table, start with the second row to allow the Table Header to be further differentiated from the the row directly beneath it.
+
+#### Benefits of striping
+- Striping makes data within the Table easier to read by increasing differentiation between rows.
+    - Ensure that nested components within striped rows continue to meet contrast accessibility criteria.
```

Comment:
Is this line advising end-users to themselves ensure that nested components meet contrast requirements? If so, I think it could be called out a bit better or clarified.

This is one case where adding a "Please" would help clarity for example I believe.
EG: "Please ensure that..."

Reply:
Now that I think about it, it's probably worthwhile to put this content in a banner.